src/opi/output/models/json/gbw/gbw_results.py

```diff
@@ -1,12 +1,12 @@
 from pydantic import Field
 
-from opi.output.models.base.get_item import GetItem
 from opi.output.models.json.gbw.properties.cite import Cite
 from opi.output.models.json.gbw.properties.header import OrcaHeader
 from opi.output.models.json.gbw.properties.molecule import Molecule
+from opi.output.models.jsonreadable import JSONLoadable
```

Comment:
```suggestion
from opi.output.models.json_loadable import JSONLoadable
```

Please pick one name here: either the module and the class both should use the term "loadable" or "readable". But don't mix them!